REFERENCE.md

```diff
@@ -322,30 +382,40 @@ Data type: `Boolean`
 
 
 
+Default value: ``false``
+
 ##### <a name="network_path"></a>`network_path`
 
 Data type: `Stdlib::Absolutepath`
 
 where all networkd files are placed in
 
+Default value: `'/etc/systemd/network'`
+
 ##### <a name="manage_accounting"></a>`manage_accounting`
 
 Data type: `Boolean`
 
 when enabled, the different accounting options (network traffic, IO, CPU util...) are enabled for units
 
+Default value: ``false``
+
 ##### <a name="accounting"></a>`accounting`
 
 Data type: `Hash[String,String]`
 
-Hash of the different accounting options. This highly depends on the used systemd version. The module provides sane defaults.
+Hash of the different accounting options. This highly depends on the used systemd version. The module provides sane defaults per operating system.
+
+Default value: `{}`
```

Comment:
This may be a bit misleading now but I'm not sure how to do it better.

Reply:
I think we should drop the default empty hash from the init.pp

Reply:
updated it